APIs_Platform/time_HelloWorld/README.md

```diff
@@ -0,0 +1,6 @@
+# Time example
+
+You can use the time interface to access the Real Time Clock (RTC). The time is set as an offset measured in seconds from the time epoch, which is library specific. The accepted time epoch is the Unix Epoch. An online converter between human readable time and Unix Epoch time is handy, try this one. If the system is not battery powered, then on each reset, the RTC time is reset. Make sure to either provide battery power to keep the time or to reset it each time the device is run.
```

Comment:
Query: When we say "try this one," what do we mean?

Reply:
My bad, I missed links.
Fixed
Also updated description a bit

Reply:
@mprse please review the description